quartz/components/styles/explorer.scss

```diff
@@ -62,7 +62,8 @@ button#explorer {
 #explorer-content {
   list-style: none;
   overflow: hidden;
-  max-height: none;
+  overflow-y: auto;
+  max-height: calc(100% - 28px);
```

Comment:
hmm is there a reason why it is 28px?

Reply:
I changed to a flex display approach to remove the need for this `calc`.